Ports/README.md

```diff
@@ -286,16 +289,24 @@ to `true`, and simply skip them otherwise.
 
 Defaults to `false`.
 
+#### `description`
+
+The description of the port. Currently required in the list of [available ports](#available-ports) only.
+
```

Comment:
I think these `Ports/README.md` changes should not have their own commit, but should be incorporated into the commits with the relevant changes instead.

Reply:
should be better now!